internal/api-key/command_list.go

```diff
@@ -51,7 +51,7 @@ func (c *command) list(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	if resourceType == resource.Cloud {
+	if resourceType == presource.Cloud || resourceType == presource.Tableflow {
```

Comment:
Potential followup: maybe Flink needs to be added here too. We should check, after.